app/controllers/storage_resource_controller.rb

```diff
@@ -19,8 +19,14 @@ def show
     super
   end
 
-  def breadcrumb_name(_model)
-    _('Storage Resources')
+  def breadcrumbs_options
+    {
+      :breadcrumbs => [
+        {:title => _("Storage")},
+        # using url_controller in :url=> here doesn't work, '/storage_resource' doesn't route to '/storage_resource/show_list' for some reason
+        {:title => _("Storage Resources"), :url => '/storage_resource/show_list'},
+      ],
```

Comment:
note the comment here..
using `{:title => _("Storage Resources"), :url => controller_url},` 
led to the error below, which seems to happen because '/storage_resource' doesn't route to '/storage_resource/show_list'.
couldn't find the bug preventing the routing, so hard-coded it instead.

the error:
```
[----] I, [2023-02-05T15:32:59.997413 #203714:15c70]  INFO -- : Started GET "/storage_resource" for 9.25.202.27 at 2023-02-05 15:32:59 +0200
[----] F, [2023-02-05T15:33:00.042269 #203714:15c70] FATAL -- :   
ActionController::RoutingError (No route matches [GET] "/storage_resource"):
  
actionpack (6.1.7.2) lib/action_dispatch/middleware/debug_exceptions.rb:33:in `call'
actionpack (6.1.7.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.7.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.7.2) lib/rails/rack/logger.rb:28:in `call'
sprockets-rails (3.4.2) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (6.1.7.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.7.2) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.6.2) lib/rack/method_override.rb:24:in `call'
rack (2.2.6.2) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.7.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.7.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.7.2) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.6.2) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.7.2) lib/action_dispatch/middleware/host_authorization.rb:148:in `call'
secure_headers (3.9.0) lib/secure_headers/middleware.rb:12:in `call'
railties (6.1.7.2) lib/rails/engine.rb:539:in `call'
railties (6.1.7.2) lib/rails/railtie.rb:207:in `public_send'
railties (6.1.7.2) lib/rails/railtie.rb:207:in `method_missing'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
```